solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -277,6 +282,8 @@ contract RandomBeacon is Ownable {
         // TODO: Revisit if initial value of 2 weeks is enough.
         sortitionPoolRewardsBanDuration = 2 weeks;
         relayEntryTimeoutNotificationRewardMultiplier = 5;
+        // TODO: Revisit the initial value.
+        minimumStake = 100e3;
```

Comment:
Let's set it to 100k T; we need e18 here.

Reply:
7325bfd